apps/src/code-studio/initApp/loadApp.js

```diff
@@ -279,7 +279,7 @@ function tryToUploadShareImageToS3({image, level}) {
 function loadProjectAndCheckAbuse(appOptions) {
   return new Promise((resolve, reject) => {
     project.load().then(() => {
-      if (project.hideBecauseAbusive() && !appOptions.canResetAbuse) {
+      if (project.hideBecauseAbusive()) {
```

Comment:
This check moved down into `showEvenIfPolicyViolatingOrAbusiveProject()` which is called by `hideBecauseAbusive()`.